.github/workflows/backport.yml

```
@@ -21,6 +22,6 @@ jobs:
         uses: ./actions/backport
         with:
           metricsWriteAPIKey: ${{secrets.GRAFANA_MISC_STATS_API_KEY}}
-          token: ${{secrets.GH_BOT_ACCESS_TOKEN}}
+          token: ${{secrets.GH_TOKEN}}
```

```suggestion
          token: ${{secrets.GITHUB_TOKEN}}
```

https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#about-the-github_token-secret

---

:man_facepalming: